e2e_playwright/latest_material_symbol.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import streamlit as st
```

Comment:
**suggestion:** Remove these e2e_playwright changes. Looks like these tests have since been moved to https://github.com/streamlit/streamlit/blob/develop/e2e_playwright/st_alert_test.py#L61